[2025-03-08T14:36:01.579+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweather_data_lake.initialize_buckets scheduled__2025-03-08T14:35:00+00:00 [queued]>
[2025-03-08T14:36:01.582+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweather_data_lake.initialize_buckets scheduled__2025-03-08T14:35:00+00:00 [queued]>
[2025-03-08T14:36:01.582+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2025-03-08T14:36:01.588+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): initialize_buckets> on 2025-03-08 14:35:00+00:00
[2025-03-08T14:36:01.596+0000] {standard_task_runner.py:57} INFO - Started process 8293 to run task
[2025-03-08T14:36:01.599+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'openweather_data_lake', 'initialize_buckets', 'scheduled__2025-03-08T14:35:00+00:00', '--job-id', '270', '--raw', '--subdir', 'DAGS_FOLDER/openweather_data_lake.py', '--cfg-path', '/tmp/tmp81q15eoe']
[2025-03-08T14:36:01.602+0000] {standard_task_runner.py:85} INFO - Job 270: Subtask initialize_buckets
[2025-03-08T14:36:01.616+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-08T14:36:01.638+0000] {task_command.py:415} INFO - Running <TaskInstance: openweather_data_lake.initialize_buckets scheduled__2025-03-08T14:35:00+00:00 [running]> on host cf46f0a3bf1a
[2025-03-08T14:36:01.676+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweather_data_lake' AIRFLOW_CTX_TASK_ID='initialize_buckets' AIRFLOW_CTX_EXECUTION_DATE='2025-03-08T14:35:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-08T14:35:00+00:00'
[2025-03-08T14:36:01.888+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'raw' already exists.
[2025-03-08T14:36:01.888+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'staging' already exists.
[2025-03-08T14:36:01.889+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'curated' already exists.
[2025-03-08T14:36:01.889+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-08T14:36:01.894+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=openweather_data_lake, task_id=initialize_buckets, execution_date=20250308T143500, start_date=20250308T143601, end_date=20250308T143601
[2025-03-08T14:36:01.923+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-08T14:36:01.936+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
